### PR TITLE
Volumize bundle output

### PR DESCRIPTION
**Pull request by thena-seer-sfg**
**NOTE: DO NOT discuss internal CommitChange information in your PR; this PR will be public.
Link back to the issue in the Tix repo when you need to do that.**

This makes the  bundle output in the docker container a volume stored in ./vendor/bundle, so that it doesn't have to reinstall everything when the dockerfile is rebuilt without caching.

# Test plan
`docker-compose up --build` and verify that ./vendor/bundle is populated

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -47,7 +42,7 @@ FROM ${BASE}
 
 ENV LANG en_US.UTF-8
 RUN apt-get update -qq \
-  && apt-get install -y libjemalloc2 tzdata libv8-dev curl default-jre git \
+  && apt-get install -y libjemalloc2 tzdata libv8-dev curl default-jre git build-essential libpq-dev \
```
Comment (wwahammy) [inline Dockerfile:45]: Why the change here to `build-essential` and `libpq-dev`?

Comment (thena-seer-sfg) [reply Dockerfile:45]: needed for bundling,  which didn't previously get performed by this image (only the builder image). Had to add these to do the runtime bundling in the CMD.

File: docker-compose.yml [hunk 2]
```diff
@@ -15,6 +15,7 @@ services:
     build: .
     volumes:
       - ${HOME}${USERPROFILE}/.netrc:/home/app/.netrc # for heroku credentials
+      - ./vendor/bundle:/usr/local/bundle
```
Comment (wwahammy) [inline docker-compose.yml:18]: Same problem on Linux here as mentioned in #952. But we'll accept it for now.

**Review by wwahammy — APPROVED** [review]
